ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/presentation/ui/landing/LandingScreen.kt

```diff
@@ -93,6 +94,19 @@ fun LandingScreen(
                     )
                 }
             )
+
+            Spacer(modifier = Modifier.size(4.dp))
+
+            CompactChip(
+                onClick = onClickValueDisplay,
+                label = {
+                    Text(
+                        stringResource(R.string.value_display_label),
```

Comment:
slider_or_stepper_value_to_display_label?

That's kind of long, so up to you but I would definitely change the name to something more descriptive so I can read this here and no exactly what it is vs. searching around the code paths to figure it out.

Reply:
I think we need more succinct name such as user_input_label or similar